dnf-behave-tests/features/plugins-third/swidtags.feature

```diff
@@ -0,0 +1,68 @@
+# the swidtags plugin is available only in Fedora
+@use.with_os=fedora__ge__30
+Feature: Smoke test for swidtags dnf plugin
+
+
+Background: Enable the swidtags plugin
+  Given I enable plugin "swidtags"
+
+
+@bz1689178
+Scenario: Run swidtags without command prints usage
+   When I execute dnf with args "swidtags"
+   Then the exit code is 2
+    And stderr is
+    """
+    usage: dnf swidtags [-c [config file]] [-q] [-v] [--version]
+                        [--installroot [path]] [--nodocs] [--noplugins]
+                        [--enableplugin [plugin]] [--disableplugin [plugin]]
+                        [--releasever RELEASEVER] [--setopt SETOPTS]
+                        [--skip-broken] [-h] [--allowerasing] [-b | --nobest] [-C]
+                        [-R [minutes]] [-d [debug level]] [--debugsolver]
+                        [--showduplicates] [-e ERRORLEVEL] [--obsoletes]
+                        [--rpmverbosity [debug level name]] [-y] [--assumeno]
+                        [--enablerepo [repo]] [--disablerepo [repo] | --repo
+                        [repo]] [--enable | --disable] [-x [package]]
+                        [--disableexcludes [repo]] [--repofrompath [repo,path]]
+                        [--noautoremove] [--nogpgcheck] [--color COLOR]
+                        [--refresh] [-4] [-6] [--destdir DESTDIR] [--downloadonly]
+                        [--comment COMMENT] [--bugfix] [--enhancement]
+                        [--newpackage] [--security] [--advisory ADVISORY]
+                        [--bz BUGZILLA] [--cve CVES]
+                        [--sec-severity {{Critical,Important,Moderate,Low}}]
+                        [--forcearch ARCH]
+                        {{sync,regen,purge}}
+    dnf swidtags: error: the following arguments are required: swidtagscmd
+    """
+
+
+@bz1689178
+Scenario: SWID tags are locally generated
+  Given I use repository "dnf-ci-fedora"
+    # enable locally generated SWID tags
+    And I create and substitute file "/etc/dnf/dnf.conf" with
+      """
+      [main]
+      gpgcheck=1
+      installonly_limit=3
+      clean_requirements_on_remove=True
+      pluginconfpath={context.dnf.installroot}/etc/dnf/plugins
+      """
```

Comment:
Every time I see this, I recall we should really do the automatic config generation... :slightly_smiling_face:

Reply:
True :)